packages/amplify-category-function/index.js

```diff
@@ -18,7 +18,36 @@ async function console(context) {
   context.print.info(`to be implemented: ${category} console`);
 }
 
+async function migrate(context) {
+  const { projectPath, amplifyMeta } = context.migrationInfo;
+  const migrateResourcePromises = [];
+  Object.keys(amplifyMeta).forEach((categoryName) => {
+    if (categoryName === category) {
+      Object.keys(amplifyMeta[category]).forEach((resourceName) => {
+        try {
+          const providerController = require(`./provider-utils/${amplifyMeta[category][resourceName].providerPlugin}/index`);
+          if (providerController) {
+            migrateResourcePromises.push(providerController.migrateResource(
+              projectPath,
```

Comment:
add context into the function parameter to be consistent, so the method does not need to use console.log(.), along with other usages of the context.

Reply:
Fixed.